cmd/pimo/main.go

```diff
@@ -109,6 +110,7 @@ There is NO WARRANTY, to the extent permitted by law.`, version, commit, buildDa
 	rootCmd.PersistentFlags().StringToStringVar(&cachesToLoad, "load-cache", map[string]string{}, "path for loading cache from file")
 	rootCmd.PersistentFlags().BoolVar(&skipLineOnError, "skip-line-on-error", false, "skip a line if an error occurs while masking a field")
 	rootCmd.PersistentFlags().BoolVar(&skipFieldOnError, "skip-field-on-error", false, "remove a field if an error occurs while masking this field")
+	rootCmd.PersistentFlags().StringToStringVar(&oneLiner, "mask", map[string]string{}, "one liner masking")
```

Comment:
I agree that order is a problem with `StringToStringVar`. 
Now the parsing is just a bit trickier. I'll get into it.

Reply:
Yes because now you need to extract the key to get the jsonpath. That could be a simple split on the first '=' character.

Or maybe by using a slightly different syntax, but i would be longer on the command line...

Reply:
The syntax is good and explicit in my opinion. And yes the best solution is to split on the first '=' (or the first '{' ).
It won't be that resilient though

Reply:
Correct syntax should be `pimo --mask "value=[{fluxUri: 'pimo://nameFR'}]"` as you can specify more masks for the same jsonpaths: `pimo --mask "value=[{add: "NewVal"},{fluxUri: 'pimo://nameFR'}]"`

Reply:
I agree, if both is possible then it is more usable, if not, OK for your proposal